Neos.Fusion/Classes/Exception/RuntimeException.php

```diff
@@ -17,21 +17,11 @@
  */
 class RuntimeException extends Exception
 {
-    /**
-     * @var string
-     */
-    protected $fusionPath;
+    private string $fusionPath;
 
-    /**
-     * @param string $message
-     * @param int $code
-     * @param \Exception $previous
-     * @param null $fusionPath
-     */
-    public function __construct($message = '', $code = 0, \Exception $previous = null, $fusionPath = null)
+    public function __construct(string $message, int $code, \Exception $previous, string $fusionPath)
```

Comment:
Is the api change to make $fusionPath non optional really intended. It makes sense but it is a little breaky.

Reply:
However i could not find any code that actually is harmed by this even in the oldest projects i found. So it is probably safe

Reply:
au contraire, not changing it will allow the fusion path at first to be empty and result in an obscure null error as we trust this value at other places to be non null -> thats why i changed it to avoid future pitfalls of us fusion runtime developers

And i also consider the runtime exception to be internal api ...

Reply:
THANK you for making sure i dont break stuff again btw ;) ❤️